src/compiler/checker.ts

```diff
@@ -24381,7 +24379,7 @@ namespace ts {
                 return resolveErrorCall(node);
             }
 
-            return resolveCall(node, callSignatures, candidatesOutArray, checkMode, isOptional);
+            return resolveCall(node, callSignatures, candidatesOutArray, checkMode, isOptional, /*fallbackError*/ undefined);

```

Comment:
these undefineds shouldn't be needed either

Reply:
Ah yeah, forgot to delete those when removing the extra param added in the original PR.

```suggestion
            return resolveCall(node, callSignatures, candidatesOutArray, checkMode, isOptional);
```